quarkchain/tools/archivesnapshot.sh

```diff
@@ -0,0 +1,20 @@
+sshpass -p $PriKey scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null u446960@u446960.your-storagebox.de:public-data/VERSION VERSION
```

Comment:
Where and when will this script be executed?

Reply:
this script is running on 65.109.90.160 every sunday